packages/manager/modules/pci/src/projects/project/notebooks/dashboard/general-information/tags/add/add.controller.js

```diff
@@ -0,0 +1,28 @@
+import Notebook from '../../../../Notebook.class';
+
+export default class PciNotebookTagsAddController {
+  /* @ngInject */
+  constructor($translate, NotebookService) {
+    this.$translate = $translate;
+    this.NotebookService = NotebookService;
+
+    this.label = Notebook.generateLabel();
+  }
+
+  $onInit() {
+    this.isLoading = false;
```

Comment:
this.isLoading is not set to true. may be its not required.

Reply:
done